.kitchen.yml

```diff
@@ -1,4 +1,5 @@
 ---
+<% if begin; File.read('/sys/fs/selinux/enforce') == '1'; rescue Errno::ENOENT => e; false; end %><% @SELinux=true %><% end %>
```

Comment:
i dont think this is wanted or needed since this well exec on travis, which isnt a current issue